providers/fips/self_test_data.inc

```diff
@@ -1733,6 +1812,20 @@ static const ST_KAT_SIGN st_kat_sign_tests[] = {
         ITM(rsa_expected_sig)
     },
 #ifndef OPENSSL_NO_EC
+    {
+        OSSL_SELF_TEST_DESC_SIGN_ED25519,
```

Comment:
should ED be under OPENSSL_NO_EC?

Reply:
Well, the EDDSA code lives in the tree under: crypto/ec/curve/curve25519.c and crypto/ec/curve448/ so I think it *is* an EC curve. I didn't want to just arbitrarily include it, so this seemed the best #define to use.

There is no OPENSSL_NO_EDDSA define so I think openssl considers these curves as part of elliptic curve crypto.